client/components/mma/shared/SupporterPlusBenefits.tsx

```diff
@@ -0,0 +1,69 @@
+import { css } from '@emotion/react';
+import { from, palette, space, textSans } from '@guardian/source-foundations';
+import { SvgTickRound } from '@guardian/source-react-components';
+
+export const SupporterPlusBenefitsSection = (props: {
+	hidden?: boolean;
+	withMargin?: boolean;
+}) => {
+	const benefitsCss = css`
+		${textSans.medium()};
+		list-style: none;
+		margin: ${props.withMargin ? `${space[5]}px 0 ${space[4]}px -4px` : 0};
```

Comment:
I think it would be better if the component didn't include any spacing. The spacing at the top and bottom is specific to the layout in the product card so would be better handled there. (Also makes things simpler when this gets reused somewhere else and needs different spacing around it.)

You probably _do_ always want to apply `margin-left: -4px` though as that removes the spacing on the edge of the tick icon so it's flush with the left-hand edge.